docker/eigenlayer/counter.sh

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
```

what does this do?

---

makes script fail fast if there is errors instead of going through whole script and then failing

---

sgtm